servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/util/KarafUtils.java

```diff
@@ -161,7 +164,11 @@ public static String getBundleFilePath(final IModule module) throws CoreExceptio
 			 */
 			@Override
 			public boolean accept(File pathname) {
-				return pathname.exists() && pathname.isFile() && (pathname.getName().toLowerCase().startsWith(module.getProject().getName().toLowerCase()) || pathname.getName().toLowerCase().startsWith(artifactId.toLowerCase())) && pathname.getName().toLowerCase().endsWith(getFileExtensionForPackaging(packaging));
+				return 	pathname.exists() && 
+						pathname.isFile() && 
+						(pathname.getName().toLowerCase().startsWith(module.getProject().getName().toLowerCase()) || 
```

Comment:
now that it is more readable, I think that we can also extract pathname.getName().toLowerCase() to be even more readable